hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFSInputStream.java

```diff
@@ -98,4 +100,24 @@ public int read(ByteBuffer buf) throws IOException {
 
     return bytesRead;
   }
+
+  /**
+   * Query the stream for a specific capability.
+   *
+   * @param capability string to query the stream support for.
+   * @return True if the stream supports capability.
+   */
+  @Override
+  public boolean hasCapability(String capability) {
+    switch (StringUtils.toLowerCase(capability)) {
+
+    /*TODO: Update this case to READBYTEBUFFER when
+     *  dependency version of Hadoop is 3.3.0 or later.
+     */
+    case "in:readbytebuffer":
```

Comment:
Would it make sense to introduce an `OzoneStreamCapabilities` interface extending Hadoop `StreamCapabilities` to hold this constant?  It could be used Ozone-wide (maybe only here and the test case), and constants could be dropped when Hadoop dependency is upgraded.